compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5967,7 +5967,7 @@ OMR::X86::TreeEvaluator::compressExpandBitsEvaluator(TR::Node *node, TR::CodeGen
    {
    TR::Node *srcNode = node->getFirstChild();
    TR::Node *maskNode = node->getSecondChild();
-   TR::Register *srcReg = cg->longClobberEvaluate(srcNode);
+   TR::Register *srcReg = cg->gprClobberEvaluate(srcNode, TR::InstOpCode::MOVRegReg());
```

Comment:
If I understand correctly, `TR::InstOpCode::MOVRegReg()` will yield `TR::InstOpCode::MOV8RegReg` in 64-bit mode and `TR::InstOpCode::MOV4RegReg` in 32-bit mode.  Should this choose based on the size of the operand node instead?